src/components/scheduling/SchedulingGoal.svelte

```diff
@@ -44,17 +70,26 @@
     <SchedulingGoalAnalysesBadge analyses={goal.analyses} {enabled} />
     <Input>
       <input
+        bind:this={schedulingGoalInput}
         bind:value={priority}
         class="st-input"
         disabled={!enabled}
         min="0"
         style:width="65px"
         type="number"
-        on:change={() => effects.updateSchedulingSpecGoal(goal.id, specificationId, { priority })}
+        on:change={() => updatePriority(priority)}
+        on:keydown={onKeyDown}
       />
+      <div class="priority-buttons">
+        <div class="up-button" on:click={() => focusInput() && updatePriority(priority - 1)}>
```

Comment:
Can we hide the up arrow and disable the up key when the priority is zero? A goal priority cannot be less than zero or an error is thrown. I'm thinking it's better to just avoid that case if we can.

Reply:
Ah yep, good call, I was wondering if we should just prevent that error but forgot to revisit. Will add that restriction.